docs/developer-docs/latest/getting-started/quick-start.md

```diff
@@ -1,150 +1,488 @@
 
-Navigate to [**COLLECTION TYPES** - **Restaurants**](http://localhost:1337/admin/plugins/content-manager/collectionType/application::restaurant.restaurant), in the left-hand menu.
+#### Add Categories
 
-- Click on **+ Add New Restaurants** button. Type `Biscotte Restaurant` in the **Name** field. Type `Welcome to Biscotte restaurant! Restaurant Biscotte offers a cuisine based on fresh, quality products, often local, organic when possible, and always produced by passionate producers.` into the **Description** field.
-- Click **Save**.
+Let's go [_Collection types > Categories_](http://localhost:1337/admin/plugins/content-manager/collectionType/application::category.category) and create 2 categories:
 
-If you navigate back [**COLLECTION TYPES** - **Restaurants**](http://localhost:1337/admin/plugins/content-manager/collectionType/application::restaurant.restaurant), you will see your restaurant listed in the entries.
+1. Click on **+ Add New Categories**.
+2. Type `French Food` in the **Name** field.
+3. On the right, in the **Restaurants (0)** dropdown, select `Biscotte Restaurant`.
+4. Click **Save**.
+5. Go back to _Collection types > Categories_, then click again on **+ Add New Categories**.  
+6. Type `Brunch` in the **Name** field, then click **Save**.
 
-## 6. Add categories to the "Category" Content Type
+![GIF: Add Categories](../assets/quick-start-guide/qsg-handson-categories.gif)
 
-Navigate to [**COLLECTION TYPES** - **Categories**](http://localhost:1337/admin/plugins/content-manager/collectionType/application::category.category).
+The `French Food` and `Brunch` categories are now listed in the [_Collection types > Categories_](http://localhost:1337/admin/plugins/content-manager/collectionType/application::category.category) view.
 
-- Click on **+ Add New Categories** button. Type `French Food` in the **Name** field. Select `Biscotte Restaurant`, on the right in the **Restaurant (0)** dropdown.
-- Click **Save**.
+#### Add a Category to a Restaurant
 
-Navigate back to [**COLLECTION TYPES** - **Categories**](http://localhost:1337/admin/plugins/content-manager/collectionType/application::category.category). You will see the **French Food** category listed in the entries.
+Go to [_Collection types > Restaurants_](http://localhost:1337/admin/plugins/content-manager/collectionType/application::restaurant.restaurant) in the left-hand menu, and click on `Biscotte Restaurant`.
 
-- Click on **+ Add New Categories** button. Type `Brunch` in the **Category** field. **DO NOT ADD** `Biscotte Restaurant` to the `Restaurants` dropdown on the right.
-- Click **Save**.
+In the right sidebar, under **Categories (1)**, click on **Add an item…** and select `Brunch`.  Click **Save**.
 
-You will see the **Brunch** category listed in the entries.
+### Step 3: Set Roles & Permissions
```

Comment:
I would add a short paragraph explaining why we have to do the following steps.
Like: to make the created content available, we need to open the API endpoint to fetch the content.
Not exactly like that but maybe with this idea.

With maybe a mention to the user guide about User&Permission.

Reply:
Not sure about linking too much from the Quick Start Guide to other parts of the doc. 
And dealing with Users & Permissions, to me, still seems a topic that is complicated — reading the docs Mégane wrote (really good docs, that's not an issue) would probably be a bit overwhelming, and I would rather not have it as a part of the first experience with the product.

Here, as you mentioned, the goal is just to open the API for requests. I can add a short sentence for this if needed.